guide/contributing/codebase-overview.md

```diff
@@ -4,14 +4,22 @@ A description of the structure of the codebase, conventions being followed, and
 
 ## Contents
 
-- [Tools](#tools)
-- [Component structure and standards](#component-structure-and-standards)
-- [Building components](#building-components)
-  - [Creating new components from scratch](#creating-new-components-from-scratch)
-  - [React & JSX practices](#react--jsx-practices)
-  - [Code style and conventions](#code-style-and-conventions)
-  - [Styling components](#styling-components)
-- [Writing tests](#writing-tests)
+- [Codebase overview](#codebase-overview)
+  - [Contents](#contents)
```

Comment:
Could you please remove `Codebase overview` and `Contents` links as they are not necessary but make the full list deeply nested.